packages/react-dom/src/__tests__/ReactDOMServerIntegrationForms-test.js

```diff
@@ -222,6 +222,41 @@ describe('ReactDOMServerIntegration', () => {
           expect(e.value).toBe('foo');
         },
       );
+
+      itRenders('a textarea with Symbol value with a warning', async render => {
```

Comment:
What about passing a function as a value? Should we test that here as well?

Also, `defaultValue`?

Reply:
Definitely. Thanks!